editor/components/block-mover/index.js

```diff
@@ -49,32 +35,42 @@ export function BlockMover( { onMoveUp, onMoveDown, isFirst, isLast, uids, block
 				className="editor-block-mover__control"
 				onClick={ isFirst ? null : onMoveUp }
 				icon={ upArrow }
-				tooltip={ __( 'Move Up' ) }
-				label={ getBlockMoverLabel(
-					uids.length,
-					blockType && blockType.title,
-					firstIndex,
-					isFirst,
-					isLast,
-					-1,
-				) }
+				label={ __( 'Move up' ) }
+				aria-describedby="editor-block-mover__up-description"
 				aria-disabled={ isFirst }
 			/>
 			<IconButton
 				className="editor-block-mover__control"
 				onClick={ isLast ? null : onMoveDown }
 				icon={ downArrow }
-				tooltip={ __( 'Move Down' ) }
-				label={ getBlockMoverLabel(
-					uids.length,
-					blockType && blockType.title,
-					firstIndex,
-					isFirst,
-					isLast,
-					1,
-				) }
+				label={ __( 'Move down' ) }
+				aria-describedby="editor-block-mover__down-description"
 				aria-disabled={ isLast }
 			/>
+			<span id="editor-block-mover__up-description" className="editor-block-mover__description">
+				{
+					getBlockMoverDescription(
+						uids.length,
+						blockType && blockType.title,
+						firstIndex,
+						isFirst,
+						isLast,
+						-1,
+					)
+				}
+			</span>
+			<span id="editor-block-mover__down-description" className="editor-block-mover__description">
+				{
+					getBlockMoverDescription(
+						uids.length,
+						blockType && blockType.title,
+						firstIndex,
+						isFirst,
+						isLast,
+						1,
+					)
+				}
+			</span>
 		</div>
 	);
 }
```

Comment:
Yes, let's use `withInstanceId` to ensure unique IDs. since two movers can be visible at the same time (hovering and selecting)